wiki-webapp/src/main/webapp/javascript/eXo/wiki/ckeditor/plugins/exportpdf/LICENSE.md

```diff
@@ -0,0 +1,18 @@
+Software License Agreement
+==========================
+
+**CKEditor 4 Export to PDF plugin** (https://ckeditor.com/ckeditor-4/)<br>
+Copyright (c) 2003-2021, [CKSource](http://cksource.com) Frederico Knabben. All rights reserved.
+
+CKEditor 4 Export to PDF plugin is licensed under a commercial license and is protected by copyright law.
```

Comment:
NOGO we can’t have a commercial plugin in Meeds nor eXo. Find an alternative of you need to export pdfs.

Reply:
Yep, we will not use the ck editor plug in for export PDF

Reply:
I removed this plugin